poc_iot_verifier/src/runner.rs

```diff
@@ -38,15 +40,16 @@ const LOADER_DB_POOL_SIZE: usize = 2 * LOADER_WORKERS;
 
 pub struct Runner {
     pool: PgPool,
-    store_path: String,
+    settings: Settings,
 }
 
 impl Runner {
-    pub async fn from_env() -> Result<Self> {
-        let pool = mk_db_pool(LOADER_DB_POOL_SIZE as u32).await?;
-        let store_path =
-            std::env::var("VERIFIER_STORE").unwrap_or_else(|_| String::from("/var/data/verifier"));
-        Ok(Self { pool, store_path })
+    pub async fn from_settings(settings: &Settings) -> Result<Self> {
+        let pool = settings.database.connect(LOADER_DB_POOL_SIZE).await?;
```

Comment:
should these be configurable settings as well?

Reply:
This relates back to the answer earlier, but there are multiple processes that each determine this value.  We could make a different database section in the config for each process if we feel like that adds clarity, but feels like overkill to me.